samples/web/content/apprtc/appwindow.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
```

Comment:
I think the UI for web and Chrome OS will be identical for the most part,
except:
1. the registration view, which should be shown/hidden dynamically anyway.
2. the contacts view, which is unclear how it works on Chrome OS.

So I think a script to avoid these two UI out of sync should be useful for
a long time.

On Wed, Jan 7, 2015 at 3:39 PM, Chuck Hays <notifications@github.com> wrote:

> In samples/web/content/apprtc/appwindow.html
> <https://github.com/GoogleChrome/webrtc/pull/349#discussion-diff-22625816>
> :
>
> > @@ -0,0 +1,51 @@
> > +<!DOCTYPE html>
>
> Yes, planning to do this, if it is useful, when we create the build script
> that packages the app. It isn't clear to me yet how similar index.html and
> appwindow.html wll be after we add all the new UI we are currently
> discussing.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleChrome/webrtc/pull/349/files#r22625816>.
>



-- 
Jiayang Liu